feature-libs/order/assets/translations/en/my-account-v2-order.i18n.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const MyAccountV2Order = {
+  myAccountV2OrderHistory: {
+    heading: 'All Orders ({{param}})',
+    item: '{{param}} Item',
+    items: '{{param}} Items',
+    totalPrice: 'Total Price: {{param}}',
+    consignmentCode: 'Consignment {{param}}',
+    statusDate: 'Last Updated: {{param}}',
+    returnProcessed: 'Return Processed: {{param}}',
+    deliveryPointOfServiceDetails: {
+      itemsToBePickUp: 'To Be Picked Up - ',
+    },
+    checkoutMode: {
+      deliveryEntries: 'To Be Shipped - ',
+    },
+    checkoutPickupInStore: {
+      heading: 'To Be Picked Up - ',
+    },
+    orderListResults: 'Orders List',
+    orderListPagination: 'Orders List pagination',
+    totalPriceLabel: 'Total Price',
+    orderPlaced: 'Order Placed On',
+    orderCodeLabel: 'Order Code',
+    consignmentDetailLabel: 'Consignment Information',
+    consignmentNumberLabel: 'Consignment Number',
+    consignmentStatusLabel: 'Consignment Status',
+    consignmentStatusDateLabel: 'Last Updated On',
+    estimateDeliveryLabel: 'Estimated Delivery Date',
+  },
+  myAccountV2OrderDetails: {
+    returnItems: 'Return Items',
+    cancelItems: 'Cancel Items',
+    downloadInvoices: 'Download Invoices',
+    viewAllOrders: 'View All Orders',
+    noInvoices: 'Invoices are not generated yet. Please come back later.',
+  },
+  myAccountView: {
```

Comment:
[MEDIUM]
It seems that somehow [the previous comment](https://github.com/SAP/spartacus/pull/17978#discussion_r1361895374) about using here `myAccountViewOrder` instead of `myAccountView` wasn't covered. Maybe the latest changes were not pushed?

Reply:
done